jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/jobs/QualysDataImporter.java

```diff
@@ -265,6 +266,9 @@ protected List<Map<String, Object>> getHostData(String uriPost, String inputXml)
                                 .collect(Collectors.toList());
                     }
                     break;
+                }else if (resp != null && "UNAUTHORIZED".equals(resp.get("responseCode"))) {
+                    //notify user code
+                    throw new UnAuthorisedException("You are not authorized to access the application through the API");
```

Comment:
UnAuthorisedException is not a run time exception. notification flow will not be triggered.